pkg/internal/mysqlcluster/defaults.go

```diff
@@ -105,8 +106,10 @@ func (cluster *MysqlCluster) SetDefaults(opt *options.Options) {
 				maxBinlogSize = min(binlogSpaceLimit/3, 1*gb)
 			}
 
-			// binlog-space-limit = totalSpace / 2
-			setConfigIfNotSet(cluster.Spec.MysqlConf, "binlog-space-limit", humanizeSize(binlogSpaceLimit))
+			if matched, err := regexp.MatchString("([a-z.]+/)*percona(:.+)?", cluster.Spec.Image); err != nil && matched {
```

Comment:
Let's define a utility function `IsPerconaImage` that will try to identify the source of the underlay image. Here is an example: https://github.com/presslabs/mysql-operator/blob/677a2ce5d364270604b8a0645e19a7df742f36f1/pkg/internal/mysqlcluster/mysqlcluster.go#L224

Also, please use `GetMysqlImage()` which computes the image based on more rules.

```suggestion
			if cluster.IsPerconaImage() {
```